app/routes/users.routes.js

```diff
@@ -34,6 +39,14 @@ module.exports = function (app) {
 
     app.get('/users/authenticate', users.authenticateToken);
 
+    app.post('/users/from-token/first-name', users.updateFirstName);
```

Comment:
This stuff doesn't have anything to do with your PR. Probably left overs from a merge? You should remove this stuff.